playbook/buildfarmer/scripted_buildfarm_day.md

```diff
@@ -4,15 +4,15 @@ This document describes the process for being a buildfarmer for a day.
 
 ## 1. Tool setup
 
-1. Pull the latest version of the buildfarm repository (this repo)
+1. Pull the latest version of the buildfarm-tools repository (this repo)
     ```
     git checkout main
     git pull origin main
     ```
-2. Update Known issues information.
+2. Update Known issues information so you work with the latest information:
 
     ```bash
-    cd ~/buildfarmer/database/scripts
+    cd ~/buildfarm-tools/database/scripts
```

Comment:
Maybe it makes sense for us to run this and update the database for the users?

Reply:
I don't understand your question :thinking:

Reply:
We can not automate github cli scripts :s